### PR TITLE
Updated commonDocReady from jquery to vanilla javascript

### PR DESCRIPTION
**Pull request by padmasreegade**
updated commonDocReady from jquery to vanilla javascript

### CODE REVIEW

File: themes/bootstrap3/js/common.js [hunk 1]
```diff
@@ -378,7 +378,7 @@ var VuFind = (function VuFind() {
    * @param {string|JQuery} container
    */
   var initResultScripts = function initResultScripts(container) {
-    let jqContainer = typeof container === 'string' ? $(container) : container;
+    let jqContainer = typeof container === 'string' ? document.querySelector(container) : container;
```
Comment (demiankatz) [inline themes/bootstrap3/js/common.js]: Is this an unintentional change? Changing the value of jqContainer seems unrelated to the rest of this pull request... and if we do this, we would both need to change the variable name (since it is no longer a jQuery container -- so the jq no longer means anything) and also verify whether all the init methods receiving jqContainer work properly with non-jQuery sets.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @padmasreegade -- see below for one question; once that's resolved, I can do hands-on testing of the remaining changes.

File: themes/bootstrap3/js/common.js (hunk 1 above)
Comment (padmasreegade) [reply themes/bootstrap3/js/common.js]: This is from one of my other changes, sorry for overseeing it!

**Review by demiankatz — APPROVED** [review]
This looks good to me, and all tests are passing.

However, before merging this, I'd be interested in hearing thoughts from @crhallberg or @EreMaijala on whether there are any timing concerns related to changing `$()` to `document.addEventListener('DOMContentLoaded', ...)`. (I don't necessarily have any reason to believe there is a problem here -- it's just the only change that I don't know the full implications of).

**Review by crhallberg — APPROVED** [review]

**Comment by demiankatz** [comment]
Thanks, @padmasreegade and @crhallberg!
